app/src/main/java/us/kikin/apps/android/bgplayer/ui/videos/VideoViewHolder.kt

```diff
@@ -3,49 +3,44 @@ package us.kikin.apps.android.bgplayer.ui.videos
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageView
-import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import coil.load
-import com.google.android.material.button.MaterialButton
-import us.kikin.apps.android.bgplayer.R
+import us.kikin.apps.android.bgplayer.databinding.ItemVideoBinding
 import us.kikin.apps.android.bgplayer.models.VideoModel
 
-class VideoViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+class VideoViewHolder(
+    private val binding: ItemVideoBinding
+) : RecyclerView.ViewHolder(binding.root) {
 
     companion object {
         fun from(parent: ViewGroup): VideoViewHolder {
-            val layoutInflater = LayoutInflater.from(parent.context)
-            val view = layoutInflater.inflate(R.layout.item_video, parent, false)
-            return VideoViewHolder(view)
+            val inflater = LayoutInflater.from(parent.context)
+            val binding = ItemVideoBinding.inflate(inflater)
```

Comment:
if you already have a parent, probably best to call `inflate(inflater, parent, false)`